core/Container/ContainerFactory.php

```diff
@@ -82,4 +82,16 @@ private function addEnvironmentConfig(ContainerBuilder $builder)
 
         $builder->addDefinitions($file);
     }
+
+    private function addPluginConfigs(ContainerBuilder $builder)
+    {
+        $plugins = Config::getInstance()->Plugins['Plugins'];
+
+        foreach ($plugins as $plugin) {
+            $file = Manager::getPluginsDirectory() . $plugin . '/plugin.php';
+            if (file_exists($file)) {
```

Comment:
Would be nice to have a cache at some point, especially re tracking.

Can we maybe rename/move the existing `Config` classes? Or check whether that file actually returns something and if not, ignore it? We will probably have this convention for a while so might be worth it